oarepo_vocabularies/ui/templates/semantic-ui/oarepo_vocabularies_ui/descendants.html

```diff
@@ -1,6 +1,7 @@
-<div><button class="test" id="test">Show descendants</button></div>
-<div class="ui container">
-  <div
-    data-invenio-search-config='{{ search_app_config(app_id="OarepoVocabularies.Search") | tojson }}'
-  ></div>
+<div style="margin-top: 20px">
```

Comment:
Instead of hardcoding margin pixels, there are pre-built utility classes (provided by [invenio semantic ui theme](https://github.com/inveniosoftware/invenio-theme/blob/master/invenio_theme/assets/semantic-ui/less/invenio_theme/theme/globals/site.overrides#L317-L368), that you can leverage here like `class="mt-20"` or  `rel-mt-2` for relative units of 2em